Apps/WebClient/src/ClientApp/app/components/timeline/medication.vue

```diff
@@ -333,19 +393,42 @@ export default class MedicationTimelineComponent extends Vue {
     }
   }
 
+  private addComment(): void {
```

Comment:
this logic should live in the comments component since it is repeated across immz